src/shared/utils/workspace.ts

```diff
@@ -54,13 +54,8 @@ export function tryProcessBazelFlutterConfig(logger: Logger, config: WritableWor
 			}
 		}
 
-		config.activateDevToolsEagerly = !!flutterConfig.devtoolsActivateScript;
-		config.dartSdkHomeLinux = makeFullPath(flutterConfig.dartSdkHome?.linux);
-		config.dartSdkHomeMac = makeFullPath(flutterConfig.dartSdkHome?.macos);
-		// This one replaces the args "global activate devtools"
-		config.devtoolsActivateScript = makeScript(flutterConfig.devtoolsActivateScript, 3);
-		// This one replaces the args "global run devtools"
-		config.devtoolsRunScript = makeScript(flutterConfig.devtoolsRunScript, 3);
+		// TODO(helin24): Check where this is used
+		config.activateDevToolsEagerly = false;
```

Comment:
IIRC this was because the activation script for internal workspaces was slow, so we did it eagerly at startup. If it's not just using normal devtools and is not slow, this field can probably be entirely removed.

Reply:
Do you mean startup of the workspace or of the app? I may leave this here for now because I want to think more about how we want to launch DevTools internally and this may still be relevant if we decide to use the internal daemon script.

Reply:
The workspace. Originally we would only try activate DevTools right before we needed to run it (which we did when the user asked for it). My understanding was that the internal activate script was too slow to run then so this flag promoted it to being run at workspace activation (so it should be ready before the user runs a debug session or tries to run DevTools).

Things have changed a little since in that now we try to _run_ DevTools eagerly too, so we can pass it to Flutter (`--devtools-server-address`) so it's possible this could be folded into that anyway (at least for Flutter apps).

Reply:
Okay I see. After taking a second look I think it makes sense to remove this `activateDevToolsEagerly` check. If I want to run DevTools eagerly it looks like I should make sure `startDevToolsServerEagerly` is set to true - does this require a change within the workspace file or elsewhere?

Reply:
`startDevToolsServerEagerly` should already be set to true for Flutter projects during activation:

https://github.com/Dart-Code/Dart-Code/blob/8ccb1ab3211d3a9c43a4e3b59da60d7dc2decb73/src/extension/extension.ts#L200-L207

It might be that your workspace isn't currently being detected as Flutter based on other comments - if that will be fixed you should get this for free, but otherwise could set it explicitly in `tryProcessBazelFlutterConfig` with the other fields.